javascript/ql/src/change-notes/2022-02-14-functionality-from-untrusted-source.md

```diff
@@ -0,0 +1,6 @@
+---
+category: newQuery
+---
+* A new query, `js/functionality-from-untrusted-source`, has been added to the query suite. It finds DOM elements
+  that load functionality from untrusted sources, like a `script`- or `iframe`-element using http-links.
```

Comment:
```suggestion
  that load functionality from untrusted sources, like a `script` or `iframe` elements using `http` links.
```

Reply:
using "that load functionality from untrusted sources, like ~~a~~ `script` or `iframe` elements using `http` links." here (no "a")